storybookjs/storybook
# Added support for the `haul` react-native packager

Issue: New Feature

## What I did

I added another cli option to use the haul cli instead of the react-native packager.

## How to test

1. create a react-native new app with a new install of storybook.

2. install haul from hauls app initializer.

3. `cp webpack.haul.js webpack.haul.storybook.js`

4. change the entry from 
```
module.exports = ({ platform }) => ({
  entry: `./index.${platform}.js`,
});
```
to
```
module.exports = ({ platform }) => ({
  entry: `./storybook/index.${platform}.js`,
});
```
5. add ` --haul webpack.haul.storybook.js` to your storybook command
6. run storybook. 

Haul should take over and do the job of the react-native packager.
